__tests__/app.test.js

```diff
@@ -178,4 +179,47 @@ describe("GET/api/articles/:article_id", () => {
         expect(res.body.message).toBe("Bad Request");
       });
   });
+  test.only("get 201 response with a new posted comment", () => {
```

Comment:
You should remove .only before pushing, otherwise we can't tell if the rest of the tests are passing :)